internal/config/social.go

```diff
@@ -11,6 +11,9 @@ type (
 	Social struct {
 		Enabled bool
 
+		OidcKey        string
```

Comment:
Oidc should be OIDC, much like HTTP

Reply:
this (config/social) is obsolete, will add to todo